breacharbiter.go

```diff
@@ -864,6 +864,13 @@ func (bo *breachedOutput) OutPoint() *wire.OutPoint {
 	return &bo.outpoint
 }
 
+// RequiredTxOut returns a non-nil TxOut if input commits to a certain
+// transaction output. This is used in the SINGLE|ANYONECANPAY case to make
+// sure any presigned input is still valid by including the output.
+func (bo *breachedOutput) RequiredTxOut() *wire.TxOut {
```

Comment:
Yet there's no impl of it in this commit that actually specifies a required output?

Reply:
Mh, follow-up. This one got large enough I felt. Added test where it is used!